src/kvilib/core/KviError.h

```diff
@@ -44,128 +44,129 @@ namespace KviError
-		OutOfAddressSpace                     =  41,
-		ConnectionRefused                     =  42,
-		KernelNetworkingPanic                 =  43,
-		ConnectionTimedOut                    =  44,
-		NetworkUnreachable                    =  45,
-		BrokenPipe                            =  46,
-		InvalidProxyAddress                   =  47,
-		RemoteEndClosedConnection             =  48,
-		InvalidIrcContextId                   =  49,
-		ErrorInLoadingModule                  =  50,
-		NoSuchModuleCommand                   =  51,
-		NoSuchModuleFunction                  =  52,
-		LeftOperandIsNotADictionaryReference  =  53,
-		RightOperandIsNotADictionaryReference =  54,
-		MissingObjectClassName                =  55,
-		NoSuchObjectClass                     =  56,
-		NoSuchObject                          =  57,
-		NoSuchObjectFunction                  =  58,
-		InvalidLeftOperand                    =  59,
-		NotEnoughParameters                   =  60,
-		IntegerParameterExpected              =  61,
-		InvalidParameter                      =  62,
-		NoSuchFile                            =  63,
-		OpenParenthesisExpected               =  64,
-		OpenBraceExpected                     =  65,
-		CantKillABuiltinClass                 =  66,
-		SocksV4LacksIPv6Support               =  67,
-		UnrecognizedProxyReply                =  68,
-		ProxyAuthFailed                       =  69,
-		ProxyNoAcceptableAuthMethod           =  70,
-		ProxyReply91RequestFailed             =  71,
-		ProxyReply92IdentFailed               =  72,
-		ProxyReply93IdentNotMatching          =  73,
-		ProxyReply01GeneralSOCKSFailure       =  74,
-		ProxyReply02ConnectionNotAllowed      =  75,
-		ProxyReply03NetworkUnreachable        =  76,
-		ProxyReply04HostUnreachable           =  77,
-		ProxyReply05ConnectionRefused         =  78,
-		ProxyReply06TTLExpired                =  79,
-		ProxyReply07CommandNotSupported       =  80,
-		ProxyReply08AddressTypeNotSupported   =  81,
-		ProxyReply09InvalidAddress            =  82,
-		InvalidPortNumber                     =  83,
-		SocketNotConnected                    =  84,
-		InsufficientResources                 =  85,
-		BindFailed                            =  86,
-		CantResolveLocalhost                  =  87,
-		UnsupportedImageFormat                =  88,
-		CantOpenFileForAppending              =  89,
-		CantOpenFileForWriting                =  90,
-		FileIOError                           =  91,
-		AcknowledgeError                      =  92,
-		CantOpenFileForReading                =  93,
-		CantSendAZeroSizeFile                 =  94,
-		MissingPopupName                      =  95,
-		ItemPopupOrSeparatorExpected          =  96,
-		SelfModificationNotAllowed            =  97,
-		FeatureNotAvailable                   =  98,
-		UnexpectedCharactersInArrayIndex      =  99,
-		UnexpectedEndInExpression             = 100,
-		UnexpectedEndInArrayIndex             = 101,
-		ProxyHttpFailure                      = 102,
+	enum Code
```

Comment:
Yep, see the commit message.
When bugs of clang-format are fixed, the alignment options can be enabled, and this will automatically come back.
I didn't surround this enum by "// clang-format off", because even though it's not very tidy, it's not very dirty either, and it's easily revertable by fixed clang-format from future.